tests/wpt/mozilla/tests/css/ol_japanese_iroha_ref.html

```diff
@@ -20,10 +18,10 @@
 </head>
 <body>
 <ol>
-    <li>&#x3044;. Gryffindor</li>
```

Comment:
Why this change? I *think* that’s what’s making `ol_japanese_iroha_a.html` fail.